docker/nginx/nginx-key.conf

```diff
@@ -76,6 +76,12 @@ http {
             proxy_pass http://things:${MF_THINGS_HTTP_PORT};
         }
 
+        location ~ ^/(identify){
```

Comment:
If you remove `identify` prefix you can merge this rule in the existing prox_pass regex for things service. 
Looks like +1 for removing prefix.